src/components/CardInfo.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../App.css';
+import PropTypes from 'prop-types';
+import { Container, Row, Col } from 'react-bootstrap';
+
+const CardInfo = ({ geolocation }) => {
```

Comment:
> Would it make sense to grab the geolocation directly from the redux store instead of passing it as a prop?

I don't get what you mean here. Please, let's have some discussion about this when next we meet. :grin: